lib/widgets/main_app_bar.dart

```diff
@@ -1,161 +1,150 @@
+                            Icons.assignment,
+                            color: Theme.of(context).primaryColor,
+                            size: 30.0,
+                          ),
+                          title: Text('Board Two'),
+                          onTap: () => print('Board Clicked')),
+                    ),
+                    Card(
+                      child: ListTile(
+                          leading: Icon(
+                            Icons.backup,
+                            color: Theme.of(context).primaryColor,
+                            size: 30.0,
+                          ),
+                          title: Text('Board Three'),
+                          onTap: () => print('Board Clicked')),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      'edit_home': CustomOverlayScreen(
+        backgroundColor: Colors.transparent,
+        content: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            SizedBox(
+
+            ),
+          ],
+        ),
+      ),
+    });
+    return Container(
+      padding: const EdgeInsets.symmetric(
+        vertical: 10.0,
+        horizontal: 10.0,
+      ),
+      color: Theme.of(context).primaryColor,
+      child: Row(
+        children: [
+          Expanded(
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                // Readability Settings
+                GestureDetector(
+                  onTap: () async {
+                    OverlayScreen().show(
+                      context,
+                      identifier: 'edit_home',
+                    );
+                  },
+                  child: Icon(
+                    Icons.filter_none,
+                    color: Colors.white,
+                    size: 25.0,
+                  ),
+                ),
+                // Board Name
+                GestureDetector(
+                  onTap: () async {
+                    OverlayScreen().show(
+                      context,
+                      identifier: 'display',
+                    );
+                  },
+                  child: RichText(
+                    text: TextSpan(
+                      children: [
+                        TextSpan(
+                          text: "Board Name",
+                          style: TextStyle(
+                            color: Colors.white,
+                            fontSize: 20.0,
+                          ),
+                        ),
+                        WidgetSpan(
+                          child: Icon(
+                            Icons.arrow_drop_down,
+                            size: 20.0,
+                            color: Colors.white,
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),

```

Comment:
What's this `RichText`, `TextSpan`, and `WidgetSpan` component do? Does Flutter have something like a dropdown widget?